grails-app/jobs/mama/ng/scheduler/SchedulerJob.groovy

```diff
@@ -0,0 +1,91 @@
+package mama.ng.scheduler
+
+import groovy.time.TimeCategory
+import groovyx.net.http.ContentEncoding
+import groovyx.net.http.ContentType
+import groovyx.net.http.HTTPBuilder
+import groovyx.net.http.HttpResponseException
+import groovyx.net.http.Method
+
+class SchedulerJob {
+    def concurrent = false
+
+    def cronParserService
+
+    static triggers = {
+        cron name:'cronTrigger', startDelay:1000, cronExpression: '0 0 * * * ?' } //execute every hour on the hour
+
+    def group = "Scheduler"
+
+    def execute() {
+        use (TimeCategory) {
+            def then = new Date() + 1.hour
```

Comment:
Sorry, should have been clearer. "resolution" == "how often we check the
schedules". Being able to configure, e.g. checking every minute.